### PR TITLE
Feature 27 devices filling surveys

### PR DESCRIPTION
**Pull request by kaeline**
https://github.com/Metatavu/oss-surveys-api-spec/issues/27

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -996,6 +996,55 @@ paths:
...
+          in: path
+          description: Device id
+          schema:
+            type: string
+            format: uuid
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/Layout"
```
Comment (anttileppa) [inline swagger.yaml]: This probably shouldn't be layout

File: swagger.yaml [hunk 2]
```diff
@@ -996,6 +996,55 @@ paths:
...
+        description: Payload
+        required: true
+      responses:
+        "200":
```
Comment (anttileppa) [inline swagger.yaml]: Should it just return 202 (accepted), the device does not really even need the data back...

File: swagger.yaml [hunk 3]
```diff
@@ -996,6 +996,55 @@ paths:
...
+          description: Created Layout
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Layout"
```
Comment (anttileppa) [inline swagger.yaml]: Remove this if you use http 202, or change to DevicePageSurveyAnswer when using 200

**Review by anttileppa — CHANGES_REQUESTED** [review]

File: swagger.yaml (hunk 1 above)
Comment (kaeline) [reply swagger.yaml]: typo, fixed now

File: swagger.yaml (hunk 2 above)
Comment (kaeline) [reply swagger.yaml]: it's copypaste typo, fixed

**Review by anttileppa — APPROVED** [review]

**Pull request merged**